test/app.spec.js

```diff
@@ -3,6 +3,10 @@ const fs = require('fs'),
   models = require('../app/models'),
   path = require('path');
 
+const emails = require('../app/services/emails');
+jest.mock('../app/services/emails');
+emails.sendWelcomeEmail.mockResolvedValue('SEND EMAIL');
```

Comment:
This is necessary here ?

Reply:
Without it the mocking only worked when running the specific test. I couldn't find another way to make this work.

Reply:
This should be only necessary in specific test suite, is repeated in test file too